packages/select/src/components/suggest/suggest2.tsx

```diff
@@ -252,6 +254,7 @@ export class Suggest2<T> extends AbstractPureComponent2<Suggest2Props<T>, Sugges
                     {...inputProps}
                     aria-autocomplete="list"
                     aria-expanded={isOpen}
+                    className={classNames(targetProps.className, inputProps.className)}
```

Comment:
this is also a bugfix -- `inputProps.className` was clobbering `targetProps.className` before, which would totally break things.